tsconfig.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
You can optionally extend an official config to clean this up a bit: https://www.npmjs.com/package/@tsconfig/node14